### PR TITLE
Deleted resource limits of agent ngt and added in memory mode example

### PR DESCRIPTION
**Pull request by hlts2**
### Description:

I deleted the resource limit of Vald-Agent-NGT to prevent OOM kill.
And I added an in-memory mode example without backup features.

### Environment:

- Go Version: 1.17.7
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.1

### Types of changes:

- [x] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by kpango — DISMISSED** [review]
LGTM

**Review by kpango — APPROVED** [review]
LGTM

**Review by kpango — APPROVED** [review]
LGTM

File: charts/vald/values/README.md [hunk 1]
```diff
@@ -8,7 +8,13 @@ This is the sample manifests to deploy Vald.
 helm install --values agent-ngt-standalone.yaml vald-cluster vald/vald
 ```
 
-## Deploy an vald-agent-ngt with backup features
+## Deploy Vald in memory mode without the backup features.
```
Comment (vankichi) [inline charts/vald/values/README.md:11]:
```suggestion
## Deploy Vald in memory mode without the backup features
```

**Review by vankichi — APPROVED** [review]
LGTM

**Comment by hlts2** [comment]
@kpango @vankichi 
Thank you for your review!!

**Pull request merged**